ci/integration/conjur/main.tf

```diff
@@ -0,0 +1,91 @@
+provider "aws" {
+ region = "us-east-1"
+}
+
+variable "conjur_ami_id" {
+  type    = "string"
+}
+
+variable "route53_zone_name" {
+  type    = "string"
+}
+
+resource "tls_private_key" "conjur_pcf_access_key" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+resource "aws_key_pair" "generated_key" {
+  key_name   = "conjur_pcf_vm_key"
+  public_key = "${tls_private_key.conjur_pcf_access_key.public_key_openssh}"
+}
+
+resource "aws_security_group" "allow_http_ssh" {
+  name        = "allow_http_ssh"
+  description = "Allow Conjur traffic"
+
+  # Allow HTTP
+  ingress {
+    from_port   = 80
+    to_port     = 80  
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  # Allow HTTPS
+  ingress {
+    from_port   = 443
+    to_port     = 443  
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  # Allow SSH remote acces
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
```

Comment:
What does this block do?

Reply:
This allows connections out from the Conjur VM. This is from a template that just allowed any connections out, though I haven't validated that it actually is needed.